test/functional/interface_rpc.py

```diff
@@ -1,12 +1,22 @@
 #!/usr/bin/env python3
-# Copyright (c) 2018 The Bitcoin Core developers
+# Copyright (c) 2018-2019 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Tests some generic aspects of the RPC interface."""
 
+from test_framework.authproxy import JSONRPCException
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import assert_equal, assert_greater_than_or_equal
 
+def expect_http_status(expected_http_status, expected_rpc_code,
```

Comment:
You could move this to `test/functional/test_framework/util.py` or even try to extend `assert_raises_rpc_error`?

Reply:
I'm not sure if that makes sense - checking the HTTP status code is a rather specific thing, which I don't expect us to do outside of this one test.  Thus I think it makes sense to keep the code here.

But if you disagree and see us checking status codes in other tests as well, I'm happy to move this to a more general place.

Reply:
Ok, it can be moved if needed.

Reply:
Yeah, that's what I think is best.  (But if anyone else feels like we should have this general right away, let me know and I will extend `assert_raises_rpc_error` to optionally check the HTTP status.)